### PR TITLE
Filter immune users aren't properly loaded into cache

### PR DESCRIPTION
**Pull request by ammongit**

### CODE REVIEW

**Comment by ammongit** [comment]
Thanks @Ma-wa-re

File: futaba/sql/models/filter.py [hunk 1]
```diff
@@ -265,6 +265,18 @@ def get_filter_immune_users(self, guild):
         self.immune_users_cache[guild].update(user_id for user_id, in result.fetchall())
         return self.immune_users_cache[guild]
 
+    def get_filter_immune_users(self, guild):
+        logger.info(
+            "Getting users about filter immunity in guild '%s' (%d)",
```
Comment (raylu) [inline futaba/sql/models/filter.py]: `'%s'` → `%r`

Comment (ammongit) [reply futaba/sql/models/filter.py]: `guild.name` is a string
Also I'm not sure if escaping the name makes a lot of sense

Comment (raylu) [reply futaba/sql/models/filter.py]: `%r` works on everything, including strs
it wouldn't escape anything unless there was a both a `'` and a `"` in the guild name (or something else that needed escaping)

**Comment by ammongit** [comment]
Oh I thought you were implying we should use %r instead of %s for objects.

This is used everywhere in the code, and since repr doesn't escape emojis
or anything I'm not sure how useful this is.

**Pull request merged**